python3-pykeepass.json

```diff
@@ -0,0 +1,106 @@
+            "sha256": "b3cb4af317d9b84f6df50f0cfa6840ba69556af637a83fd971537823e13d601a"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/ca/63/139b710671c1655aed3b20c1e6776118c62e9f9311152f4c6031e12a0554/lxml-4.2.4.tar.gz",
+            "sha256": "e2afbe403090f5893e254958d02875e0732975e73c4c0cdd33c1f009a61963ca"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/a0/b0/a4e3241d2dee665fea11baec21389aec6886655cd4db7647ddf96c3fad15/python-dateutil-2.7.3.tar.gz",
+            "sha256": "e27001de32f627c22380a688bcc43ce83504a7bc5da472209b4c70f02829f0b8"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/91/fa/5274ba70e4d1f0b72f49d2f87028fff0b958d709df0a94eb4cd5d0723a6e/easypysmb-1.4.3.tar.gz",
+            "sha256": "3e84a71140cf654fba1ab4b78aae2f1c29b48735074704480f42a7d0eca938ea"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/e7/a7/4cd50e57cc6f436f1cc3a7e8fa700ff9b8b4d471620629074913e3735fb2/cffi-1.11.5.tar.gz",
+            "sha256": "e90f17980e6ab0f3c2f3730e56d1fe9bcba1891eeea58966e89d352492cc74f4"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/31/9b/d52a356fa5a58685868cb6bf812457314780967d9943ce44f07021588edd/argon2_cffi-18.3.0.tar.gz",
+            "sha256": "003f588de43a817af6ecc1c06103fa0801de63849db3cb0f37576bb2da29043d"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/8c/2d/aad7f16146f4197a11f8e91fb81df177adcc2073d36a17b1491fd09df6ed/pycparser-2.18.tar.gz",
+            "sha256": "99a8ca03e29851d96616ad0404b4aad7d9ee16f25c9f9708a11faf2810f7b226"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/eb/f3/67579bb486517c0d49547f9697e36582cd19dafb5df9e687ed8e22de57fa/Mako-1.0.7.tar.gz",
+            "sha256": "4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/19/c0/f054941fa33d14378de66d2c0477d31f7ad97aa2e298a5771a7b20bc2039/construct-2.9.45.tar.gz",
+            "sha256": "2271a0efd0798679dea825ff47e22a4c550456a5db0ba8baa82f7eae0af0118c"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/c4/09/c8b11bfd95da6ca9ad731847d5a0bda42281e520bb7f72108f29c2dca5e8/pykeepass-3.0.2.tar.gz",
+            "sha256": "20b53d10e9c319301929f029579623562c815c4b4c787bf96d4bd7680581d6cd"
+        }
+    ],
+    "modules": [
+    	"python3-MarkupSafe.json",
+        "python3-setuptools_scm.json",
+        {
+            "name": "libxml2",
+            "buildsystem": "autotools",
+            "make-install-args": ["--ignore-errors"],
```

Comment:
Just curious why is the erroring and why should we ignore them?

Reply:
Because of this:
`
/usr/bin/install: cannot create regular file '/usr/lib/python2.7/site-packages/drv_libxml2.py': Read-only file system
`
`
/usr/bin/install: cannot create regular file '/usr/lib/python2.7/site-packages/libxml2.py': Read-only file system
`

Reply:
You should disable python2 installation instead or set the correct prefix if you use it.

Reply:
Thanks, Done